app/main.py

```diff
@@ -1,2 +1,120 @@
+from __future__ import annotations
+
+from typing import Iterable
+from typing import Hashable, Any
+
+
+INITIAL_CAPACITY = 8
+RESIZE_THRESHOLD = 0.66
+CAPACITY_MULTIPLIER = 2
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = INITIAL_CAPACITY
+        self.size = 0
+        self._load_factor_calculation()
+        self.data = []
```

Comment:
not really good to set this value to an empty list. Can you use one of your methods here?